app/models/taxonomy/expanded_taxonomy.rb

```diff
@@ -118,9 +118,9 @@ def build_parent_expansion
     end
 
     def build_child_expansion
-      @child_expansion = GovukTaxonomyHelpers::LinkedContentItem.from_publishing_api(
-        content_item: root_content_item,
-        expanded_links: root_expanded_links
+      @child_expansion = GovukTaxonomyHelpers::LinkedContentItem.from_content_id(
+        content_id: @content_id,
+        publishing_api: Services.publishing_api,
```

Comment:
It feels weird that you have to pass in a publishing_api object. 

It might be worth refactoring `from_content_id ` at some point so that you don't have to do that.